web/src/modules/app/containers/__tests__/__snapshots__/RouteContentSwitcher.spec.js.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`RouteContentSwitcher should render and match snapshot 1`] = `
-<Translate(Connect(CategoriesPage))
+<Connect(Translate(CategoriesPage))
```

Comment:
Question: Is the order of connect and translate relevant?

Reply:
No, not relevant. I don't know why I changed the order, probably just copy & paste?

Reply:
Just wanted to know whether that is something to actively think about ^.^